src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -127,31 +129,31 @@ def load_arguments(self, _):
         c.argument('key', help='Key to be showed.')
-        c.argument('label', help="If no label specified, show entry with null label. Does NOT support filters like other commands")
+        c.argument('label', help="If no label specified, show entry with null label. Does NOT support filters like list command")
 
     with self.argument_context('appconfig kv list') as c:
         c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
-        c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported.")
+        c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported. Use \\0 for null label.")
 
     with self.argument_context('appconfig kv restore') as c:
         c.argument('key', help='If no key specified, restore all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
         c.argument('label', help="If no label specified, restore all key-value pairs with all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported.")
 
     with self.argument_context('appconfig kv lock') as c:
         c.argument('key', help='Key to be locked.')
-        c.argument('label', help="If no label specified, lock entry with null label. Does NOT support filters like other commands")
+        c.argument('label', help="If no label specified, lock entry with null label. Does NOT support filters like list command")
 
     with self.argument_context('appconfig kv unlock') as c:
         c.argument('key', help='Key to be unlocked.')
-        c.argument('label', help="If no label specified, unlock entry with null label. Does NOT support filters like other commands")
+        c.argument('label', help="If no label specified, unlock entry with null label. Does NOT support filters like list command")
 
     with self.argument_context('appconfig revision list') as c:
         c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
-        c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported.")
+        c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported. Use \\0 for null label.")
```

Comment:
For \\0, it will be better to give single quote like '\\0'.

Reply:
Fixed.

---
In reply to: [347081357](https://github.com/Azure/azure-cli/pull/11036#discussion_r347081357) [](ancestors = 347081357)

Reply:
Could you use `Use '\\0' for null label` here to make it more obvious as a label value if possible?

Reply:
Already fixed in latest commit. You may still review the older one.